structure/linkedlist/doubly_test.go

```diff
@@ -63,15 +63,15 @@ func TestDoubly(t *testing.T) {
 
 	t.Run("Test DelAtBeg", func(t *testing.T) {
 		want := 3
-		got := newList.DelAtBeg()
+		got, _ := newList.DelAtBeg()
```

Comment:
Since you changed the signature, I would expect this to be tested as well.

Reply:
Agreed